### DISCUSSION TITLE
PGDebugger plugin

### DISCUSSION THREAD

**Discussion by segjerdingen** [dbeaver/dbeaver#16123] [General]
Is it possible to use this plugin on a Windows installation of dbeaver?  I only saw Ubuntu instructions.  Thanks!

**Comment by mkgrgis** [comment]
PostgreSQL on Windows is very strange configuration, but possible... If server of PostgreSQL have postgresql-{X}-pldebugger functions (some .so files), no problems to try DBeaver as OS-independent client for debugging.
